assured-workloads-example/.gitignore

```diff
@@ -0,0 +1 @@
+Makefile
```

Comment:
why do we need this `.gitignore`?

Reply:
I'm using this in my local folder.

Reply:
The -f param works. I'll remove this file.